src/test/java/spleetwaise/transaction/logic/commands/AddCommandTest.java

```diff
@@ -21,12 +21,11 @@
 
 public class AddCommandTest {
```

Comment:
@rollingpencil Might want to refactor AddCommandTest to utilise `TransactionBuilder.java`